exporter/exporter.go

```diff
@@ -55,11 +57,12 @@ func exporterFuncMap(cat1Template *template.Template, vsMap models.ValueSetMap)
 		"isNil":                                      isNil,
 		"derefBool":                                  derefBool,
 		"emptyMdc":                                   models.EmptyMdc,
+		"newRecordTarget":                            newRecordTarget,
 	}
 }
 
-//export GenerateCat1
-func GenerateCat1(patient []byte, measures []byte, valueSets []byte, startDate int64, endDate int64, qrdaVersion string) string {
+// GenerateCat1 generates a cat1 xml string for export
```

Comment:
Not in this library, though. The true bindings are in go-cda-tools. This function gets imported like a normal go library into that package where it then is called by the ruby code. That's where the export comment is needed and is.